lib/src/sdk_env.dart

```diff
@@ -492,8 +492,9 @@ class DartdocResult {
 }
 
 class DartSdkInfo {
-  static final _sdkRegexp =
-      RegExp('Dart VM version:\\s([^\\s]+)\\s\\(([^\\)]+)\\) on "(\\w+)"');
+  static final _sdkRegexp = RegExp(
+    r'Dart VM version:\s([^\s]+)\s\(([^\)]+)\)(\s\(([^\)]+)\))? on "(\w+)"',
```

Comment:
I think you need non capturing groups, or the date will no longer be match[3].
```
r'Dart VM version:\s([^\s]+)\s\(([^\)]+)\)(?:\s\((?:[^\)]+)\))? on "(\w+)"'
```

Reply:
We'll also need to handle both the old and the new output (e.g. try the longer/new regex first, then fallback to the old one), or, if this is a breaking change, we'll need to be specific about it.

Reply:
I do handle the optional 2nd item – there's a ? in there that's hard to read.

This works on the bots (without the new change) and on my machine (with the change).

A test would be good, though.